static/css/modules/_patterns_testimonials.scss

```diff
@@ -0,0 +1,156 @@
+// Pull quote styling
+@mixin vf-p-testimonials {
+  %blockquote {
+    border: 0;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    height: 100%;
+    padding-left: 2rem;
+    padding-right: 1.25rem;
+    position: relative;
+  }  
+  
+  %image {
+    align-self: center;
+    border-radius: 50%;
+    margin-bottom: 1rem;
+    max-width: 70px;
+  }
+  
+  %quotes {
+    color: $brand-color;
+    display: inline-block;
+    font-size: 2rem;
+    font-weight: bold;
+    line-height: 1rem;
+    max-width: 1.25rem;
+  }
+  
+  %quote {
+    align-self: flex-start;
+    display: inline-block;
+    font-size: 1.125rem;
+    font-style: normal;
+    margin: 0 0 auto;
+    padding-bottom: 2rem;
+    padding-right: 1rem;
+  
+    &:first-of-type::before {
+      @extend %quotes;
+      content: '\201C\2002'; // Unicode for left double quotation mark +  1/2 em space
+      margin-left: -1.5rem;
+      margin-right: .25rem;
+    }
+
+    &:last-of-type {
+      margin-bottom: 0;
+    }
+
+    &:last-of-type::after {
+      @extend %quotes;
+      content: '\2002\201E'; // Unicode for 1/2 em space + low double quotation mark
+      display: inline-block;
+      line-height: 0;
+      margin-left: .7rem;
+      position: relative;
+      top: -.5rem;
+    }
+  }
+  
+  %citation-wrap {
+    align-self: flex-end;
+    display: flex;
+    line-height: 1.4;
+    margin-top: auto;
+    min-height: 72px;
+    width: 100%;
+  }
+  
+  %citation {
+    align-items: stretch;
```

Comment:
It's better to try and only have common styles in placeholders so we don't end up with styles that override each other. Moving this style to the `&__citation` would give us this.